apps/main-app/src/routes.js

```diff
@@ -52,14 +52,6 @@ export default (
         }}
```

Comment:
Im guessing you can remove these other routes also? Given external=true?

Reply:
On my local machine the api works fine this way, but I get a page not found with Alliance Mine.

Reply:
Thats fine... just trying to be consistent with the links.

Reply:
There is proxies also setup in the UI... so it might be good to check them also for alliance mine... that is why the API is working its being proxyed

Reply:
Do you mean the dev server proxies in the webpack.config?